src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -26,7 +27,7 @@
  *
  * @final since version 3.4
```

Comment:
https://github.com/symfony/symfony/pull/23624#discussion_r131486828 it's done for allowing proxiex/mocks solely.

> extending internally is not an issue to me

For most commands i dont see it happen, yet we made containerdebug internal because of this :) and commands in components/bridges are also not marked final.

Last, some commands were already final for real.